bindings/ros/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 2.8.3)
+
+if(UNIX)
+
+	#set variables
+	if(NOT DEFINED CATKIN_WS)
+		message("-- Creating catkin workspace in build directory")
```

Comment:
Are these messages useful for user? Or they acted as debug messages while writing and testing these cmake instructions?

Reply:
Generally, ros packages are generated in a catkin workspace, so we decided that there are two use cases for the ros bindings:
1.A catkin workspace directory is created in the build directory of aditof_sdk 
2.The user has already created a catkin workspace
For the second case, the user must provide the path of the workspace using the CATKIN_WS variable. The messages were meant for users, to make it easier for them to check what is the path to the catkin workspace used, maybe warn them that they forgot to set the parameter. What is your opinion? Are the messages useful or not?

Reply:
In case 1. things are straightforward, as one can see that everything (including the ros packages) is created in the build folder.
In case 2. the user provides the path. Printing the path that user provided isn't of much help. If user forgets to provide path, the cmake should fail as it can't continue anymore, right?

Reply:
You are right, but how to tell if the user didn't provide a path because he wants to follow the first case or because he wants to follow the second case and forgot? First option, the cmake shouldn't fail, but the second should. Would it be ok for the user to set the CATKIN_WS variable to "default" or something similar for the first case and an unset variable would result in an error message and cmake failing?

Reply:
What if user forgets to unset?
You should detect automatically if you're in case 1 or case 2.

Reply:
As discussed, the second case is no longer supported. Fixed in commit https://github.com/analogdevicesinc/aditof_sdk/pull/241/commits/12add7a2dcb99d8987e9e3b1573f21e1e1957c76